src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
```

Comment:
The SCSS file should be imported with the correct extension. Change './styles/main.scss' to './styles/main.css'.